pkg/druid.go

```diff
@@ -792,7 +796,9 @@ func (ds *druidDatasource) prepareResponse(resp *druidResponse, settings map[str
 				}
 			}
 		}
-		frame.Fields = append(frame.Fields, data.NewField(c.Name, nil, ff))
+    if hideEmptyColumns, ok := settings["hideEmptyColumns"]; ok && (!hideEmptyColumns.(bool) || !columnIsEmpty) {
+      frame.Fields = append(frame.Fields, data.NewField(c.Name, nil, ff))
+    }
```

Comment:
the `ok` check here means we cannot append columns if the `"hideEmptyColumns"` property isn't set in `settings`.
Here it works because we set it to `false` by default in the frontend side but if we change it it'll break this part.

I suggest to set once `"hideEmptyColumns"` at the beginning of the function:

```go
frame := data.NewFrame("response")
hideEmptyColumns, _ := settings["hideEmptyColumns"].(bool) // if not found in map the boolean will be false
...
for ic, c := range resp.Columns {
  ...
  // then check if column has to be hide
  if (hideEmptyColumns && columnIsEmpty) {
	  		continue
  }
  frame.Fields = append(frame.Fields, data.NewField(c.Name, nil, ff))
}
```
imo it's clearer to let the default choice (append column) outside the `if`. What do you think ?
(ps: looks like the format is broken by spaces here vs tabs in all codebase, could you run gofmt please ?)

Reply:
sounds good to me. made the requested changes and ran gofmt